BREAKING.md

```diff
@@ -1,3 +1,8 @@
+## 0.46 Breaking changes
+
```

Comment:
The "`@fluid-experimental/tinylicious-client` package name changed" header needs to be linked at the top here, similar to 0.45 Breaking changes